kore/src/Kore/Exec.hs

```diff
@@ -213,14 +209,15 @@ exec
         , MonadProf smt
         )
     => Limit Natural
+    -> Limit Natural
     -> VerifiedModule StepperAttributes
     -- ^ The main module
-    -> ([Rewrite] -> [Strategy (Prim Rewrite)])
-    -- ^ The strategy to use for execution; see examples in "Kore.Step.Step"
+    -> ExecutionMode
+    -- ^ The execution mode
```

Comment:
I don't think this documentation is helpful. The reader can always jump to the documentation for `ExecutionMode`!

```suggestion
```